src/collective/easyform/browser/widget.pt

```diff
@@ -0,0 +1,44 @@
+<div
+   metal:define-macro="widget-wrapper"
+   i18n:domain="plone"
+   tal:define="widget nocall:context;
+               hidden python:widget.mode == 'hidden';
+               error widget/error;
+               error_class python:error and ' error' or '';
+               empty_values python: (None, '', [], ('', '', '', '00', '00', ''), ('', '', ''));
+               empty_class python: (widget.value in empty_values) and ' empty' or '';
+               wrapper_css_class  widget/wrapper_css_class|nothing;
+               fieldname_class string:kssattr-fieldname-${widget/name};
+               dependencies view/@@dependencies;
+               pat_depends_class python: dependencies and 'pat-depends'"
+   data-pat-inlinevalidation='{"type":"z3c.form"}'
+   data-pat-depends="${dependencies}"
+   tal:attributes="class string:field pat-inlinevalidation ${fieldname_class}${error_class}${empty_class} ${wrapper_css_class} ${view/@@css_class} ${pat_depends_class};
+                   data-fieldname widget/name;
+                   id string:formfield-${widget/id};">
+        <label for="" class="horizontal"
+            tal:attributes="for widget/id"
+            tal:condition="python: not hidden and widget.label">
+            <span i18n:translate="" tal:replace="widget/label">label</span>
+
+            <span class="required horizontal" title="Required"
+                tal:condition="python:widget.required and widget.mode == 'input'"
+                i18n:attributes="title title_required;">&nbsp;</span>
+
+            <span class="formHelp"
+                tal:define="description python: getattr(widget, 'description', widget.field.description)"
+                i18n:translate=""
+                tal:content="structure description"
+                tal:condition="python:description and not hidden"
+                >field description
+            </span>
+        </label>
+
+        <div class="fieldErrorBox"
+            tal:content="structure error/render|nothing">
+            Error
+        </div>
+
+        <input type="text" tal:replace="structure widget/render"
+            metal:define-slot="widget" />
+</div>
```

Comment:
This template doesn't match the current BS5 implementation here: https://github.com/plone/plone.app.z3cform/blob/master/plone/app/z3cform/templates/widget.pt

I assume, that this branch is Plone 6 only ...